### PR TITLE
docs(react-grid): add reference for createRowCache function

### PR DESCRIPTION
**Pull request by ushkal**

### CODE REVIEW

File: packages/dx-react-grid/docs/reference/create-row-cache.md [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+Name | Type | Default | Description
+-----|------|---------|------------
+pageSize | number | 100 | The number of rows on virtual page. It must be equal to the value of pageSize property of the [VirtualTableState](virtual-table-state.md#properties) plugin.
+capacity | number | Infinity | The maximum number of pages that cache should store. When the reaches its capacity, it discards the least recently used virtual page.
+
+### Return value
+
+Type | Description
+-----|------------
+RowCache | A row cache.
```
Comment (MaximKudriavtsev) [inline packages/dx-react-grid/docs/reference/create-row-cache.md]: Can we add a reference to RowCache interface?
Can we add a section `How to import`?

Comment (ushkal) [reply packages/dx-react-grid/docs/reference/create-row-cache.md]: It's a good idea to add a usage example.
The RowCache interface described below, I think it's better to keep them in a single document

Comment (MaximKudriavtsev) [reply packages/dx-react-grid/docs/reference/create-row-cache.md]: No, I mean other points.

- Add [RowCache](`#RowCache`) link.
- Add import section as we did here
image

File: packages/dx-react-grid/docs/reference/create-row-cache.md [hunk 2]
```diff
@@ -0,0 +1,38 @@
+# createRowCache
+
+A function that instantiates a default implementation of row cache. The cache is advised to be used in [lazy load mode](../guides/lazy-loading.md) to reduce the number of network requests.
```
Comment (RomanTsukanov) [inline packages/dx-react-grid/docs/reference/create-row-cache.md]:
```suggestion
A function that instantiates the default implementation of row caching. We recommend that you use row caching in [lazy loading mode](../guides/lazy-loading.md) to reduce the number of network requests.
```

File: packages/dx-react-grid/docs/reference/create-row-cache.md [hunk 3]
```diff
@@ -0,0 +1,38 @@
...
+import { createRowCache } from '@devexpress/dx-react-grid';
+```
+
+## User reference
+
+### Arguments
+
+Name | Type | Default | Description
+-----|------|---------|------------
+pageSize | number | 100 | The number of rows on virtual page. It must be equal to the value of pageSize property of the [VirtualTableState](virtual-table-state.md#properties) plugin.
```
Comment (RomanTsukanov) [inline packages/dx-react-grid/docs/reference/create-row-cache.md]:
```suggestion
pageSize | number | 100 | The number of rows on a virtual page. It should be equal to the `pageSize` property value specified in the [VirtualTableState](virtual-table-state.md#properties) plugin.
```

File: packages/dx-react-grid/docs/reference/create-row-cache.md [hunk 4]
```diff
@@ -0,0 +1,38 @@
...
+capacity | number | Infinity | The maximum number of pages that cache should store. When the cache reaches its capacity, it discards the least recently used virtual page.
+
+### Return value
```
Comment (RomanTsukanov) [inline packages/dx-react-grid/docs/reference/create-row-cache.md]:
```suggestion
### Return Value
```

File: packages/dx-react-grid/docs/reference/create-row-cache.md [hunk 5]
```diff
@@ -0,0 +1,38 @@
...
+```
+
+## User reference
+
+### Arguments
+
+Name | Type | Default | Description
+-----|------|---------|------------
+pageSize | number | 100 | The number of rows on virtual page. It must be equal to the value of pageSize property of the [VirtualTableState](virtual-table-state.md#properties) plugin.
+capacity | number | Infinity | The maximum number of pages that cache should store. When the cache reaches its capacity, it discards the least recently used virtual page.
```
Comment (RomanTsukanov) [inline packages/dx-react-grid/docs/reference/create-row-cache.md]:
```suggestion
capacity | number | Infinity | The maximum number of virtual pages that the cache should store. When the capacity is reached, the least recently used page is removed from the cache to make room for new pages.
```

File: packages/dx-react-grid/docs/reference/create-row-cache.md [hunk 6]
```diff
@@ -0,0 +1,38 @@
...
+
+## Interfaces
+
+### RowCache
+
+Describes a row cache.
+
+Field | Type | Description
+------|------|------------
+getRows | (skip: number, take: number) => any[] | Returns `take` number of rows starting with the `skip` row from the cache.
```
Comment (RomanTsukanov) [inline packages/dx-react-grid/docs/reference/create-row-cache.md]:
```suggestion
getRows | (skip: number, take: number) => any[] | Returns from the cache the `take` number of rows starting with the `skip` row.
```

File: packages/dx-react-grid/docs/reference/create-row-cache.md [hunk 7]
```diff
@@ -0,0 +1,38 @@
...
+setRows | (skip: number, rows: ReadonlyArray&lt;any&gt;) => void | Adds `rows` to the cache.
+invalidate | () => void | Clears the cache
```
Comment (RomanTsukanov) [inline packages/dx-react-grid/docs/reference/create-row-cache.md]:
```suggestion
invalidate | () => void | Clears the cache.
```

File: packages/dx-react-grid/docs/reference/create-row-cache.md [hunk 8]
```diff
@@ -0,0 +1,38 @@
...
+```
+
+## User reference
+
+### Arguments
+
+Name | Type | Default | Description
+-----|------|---------|------------
+pageSize | number | 100 | The number of rows on a virtual page. It should be equal to the `pageSize` property value specified in the [VirtualTableState](virtual-table-state.md#properties) plugin.
+capacity | number | Infinity | The maximum number of virtual pages that the cache should store. When the capacity is reached, the least recently used page is removed from the cache to make room for new pages.
```
Comment (arminal) [inline packages/dx-react-grid/docs/reference/create-row-cache.md]:
```suggestion
capacity | number | Infinity | The maximum number of virtual pages that the cache should store. When capacity is reached, the oldest used page is removed from the cache to make room for new pages.
```

**Pull request merged**